adhocracy4/actions/signals.py

```diff
@@ -1,11 +1,19 @@
+from itertools import chain
 from django.apps import apps
 from django.conf import settings
-from django.db.models.signals import post_save
+from django.contrib.contenttypes.models import ContentType
+from django.db.models.signals import post_delete, post_save
 
 from .models import Action
 from .verbs import Verbs
 
 
+SYSTEM_ACTIONABLES = (
```

Comment:
if it is not created due to a setting in A4_ACTIONABLES but due to a call of the management function 'create_system_actions'